pt-BR/mail/chrome/messenger/preferences/compose.dtd

```diff
@@ -43,103 +43,103 @@
 <!ENTITY directories.accesskey                 "S">
 <!ENTITY editDirectories.label                 "Editar diretórios…">
 <!ENTITY editDirectories.accesskey             "E">
-<!ENTITY matchText.label                       "If there is a match in your local address books:">
-<!ENTITY skipDirectory.label                   "Do not search in the directory">
+<!ENTITY matchText.label                       "Se já houver um nos seus catálogos locais de endereços:">
+<!ENTITY skipDirectory.label                   "Não pesquisar no diretório">
 
 <!ENTITY sendOptions.label                     "Formato de envio…">
 <!ENTITY sendOptions.accesskey                 "m">
 
-<!ENTITY attachmentReminder.label              "Check for missing attachments">
+<!ENTITY attachmentReminder.label              "Verificar anexos ausentes">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY attachmentReminder.label              "Verifique se anexou todos os arquivos">